pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java

```diff
@@ -322,16 +342,16 @@ private void stopRunningEnvironment(Throwable ex) {
         if(ex instanceof BuildProcessException) {
             BuildProcessException bpEx = (BuildProcessException) ex;
             try {
-                if (bpEx.getRunningEnvironment() != null)
-                    bpEx.getRunningEnvironment().destroyEnvironment();
+                if (bpEx.getDestroyableEnvironmnet() != null)
+                    bpEx.getDestroyableEnvironmnet().destroyEnvironment();
             } catch (EnvironmentDriverException envE) {
-                log.warn("Running environment" + bpEx.getRunningEnvironment() + " couldn't be destroyed!", envE);
+                log.warn("Running environment" + bpEx.getDestroyableEnvironmnet() + " couldn't be destroyed!", envE);
```

Comment:
I think it is warning, because it causes leaks, but the application can run good in user's point of view.